/udp_bridge:
  ros__parameters:
    ip_addr: "127.0.0.1"
    local_port: 6000
    remote_port: 6000
    packet_size: 1024
    timeout: 1.0
    url: "package://udp_bridge/config/channels.yaml"